manifests/install/charts/as-a-second-scheduler/templates/rbac.yaml

```diff
@@ -0,0 +1,86 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: scheduler-plugins-scheduler
+rules:
+- apiGroups: ["scheduling.sigs.k8s.io"]
+  resources: ["podgroups", "elasticquotas"]
+  verbs: ["get", "list", "watch", "create", "delete", "update", "patch"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: scheduler-plugins-scheduler-as-admin
```

Comment:
Can we use one set of role/rolebinding for the secondary scheduler setup? I don't quite like to define 3 ClusterRoleBindings here: xxx-as-admin, xxx-as-kube-scheduler and xxx-as-volume-scheduler.

Reply:
combined to one clusterrole